src/test/java/Tests/SignInTest.java

```diff
@@ -0,0 +1,16 @@
+package Tests;
+
+import Steps.Steps;
+import org.testng.annotations.Test;
+
+public class SignInTest extends BaseTest{
+
+@Test
+    public void test() {
```

Comment:
The same - The name of test should be readable. I should understand what this test do when I read the name

Reply:
renamed